.circleci/config.yml

```
@@ -6,12 +6,22 @@ workflows:
   package-and-push-chart-on-tag:
     jobs:
       - architect/push-to-app-catalog:
-          context: "architect"
-          name: "package and push efk-stack-app chart"
-          app_catalog: "giantswarm-catalog"
-          app_catalog_test: "giantswarm-test-catalog"
-          chart: "efk-stack-app"
+          name: package and push efk-stack-app chart
+          chart: efk-stack-app
+          app_catalog: giantswarm-catalog
+          app_catalog_test: giantswarm-test-catalog
+          executor: app-build-suite
           # Trigger job on git tag.
           filters:
             tags:
               only: /^v.*/
+
+      - architect/run-tests-with-abs:
+          name: execute chart tests
+          resource_class: 2xlarge
+          chart_dir: ./helm/efk-stack-app
+          # app-build-suite_version: "v0.1.8"
+          app-build-suite_container_tag: "0.1.8"
```

Don't mind me, just testing something

```suggestion
          app-build-suite_version: "210d995d3e455207dcd0ebdf84152847cdcdae3b"
          app-build-suite_container_tag: "0.1.8-210d995d3e455207dcd0ebdf84152847cdcdae3b"
```